enterprise/server/gcplink/gcplink.go

```diff
@@ -0,0 +1,83 @@
+package gcplink
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/enterprise/server/util/keystore"
+	"github.com/buildbuddy-io/buildbuddy/server/environment"
+	"github.com/buildbuddy-io/buildbuddy/server/util/cookie"
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+
+	ctxpb "github.com/buildbuddy-io/buildbuddy/proto/context"
+	skpb "github.com/buildbuddy-io/buildbuddy/proto/secrets"
+	requestcontext "github.com/buildbuddy-io/buildbuddy/server/util/request_context"
+)
+
+const (
+	Issuer                      = "https://accounts.google.com"
+	scope                       = "https://www.googleapis.com/auth/cloud-platform"
+	paramName                   = "link_gcp_for_group"
+	cookieName                  = "link-gcp-for-group"
```

Comment:
lol I guess i'm pedantic but i'd call these linkParamName and linkCookieName just so the code below is a little clearer (yeah i know it's in the gcplink package but it's still a little disorienting to read GetCookie(r, cookieName) and be like... what is cookieName haha)

Reply:
Done